assets/scss/local.scss

```diff
@@ -24,6 +24,10 @@
   color: govuk-colour("red");
 }
 
+.tag-margin-bottom {
+  margin-bottom: 10px;
+}
```

Comment:
I would just use `govuk-!-margin-bottom-2` instead in the one place where you're using `tag-margin-bottom` as it's basically doing the same thing but without the need to define an extra class.